<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="ids">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="maxVisibleItems">
           <number>5</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAtBottom</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToMinimumContentsLength</enum>
          </property>
          <property name="minimumContentsLength">
           <number>0</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="plotBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="plotAllBtn">
          <property name="text">
           <string>Plot all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>List Ponts</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Gate definer</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTabWidget" name="Gate_Tab">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>102</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>Point 1</string>
               </attribute>
               <widget class="QLabel" name="label_8">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>31</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLabel" name="label_9">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>40</y>
                  <width>31</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Y</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLineEdit" name="Gate_P1_x">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>10</y>
                  <width>61</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="Gate_P1_y">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>40</y>
                  <width>61</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </widget>
              <widget class="QWidget" name="tab_4">
               <attribute name="title">
                <string>Point 2</string>
               </attribute>
               <widget class="QLineEdit" name="Gate_P2_x">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>10</y>
                  <width>91</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="Gate_P2_y">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>40</y>
                  <width>91</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_10">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>31</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLabel" name="label_11">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>40</y>
                  <width>31</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Y</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdVal">
              <property name="text">
               <string>Validate</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>200</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="reelTimeCheck">
              <property name="text">
               <string>Plot by time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEdit">
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QTableWidget" name="Table_Point"/>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>100</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="tb_id">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>52</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::WheelFocus</enum>
             </property>
             <property name="maxVisibleItems">
              <number>5</number>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::InsertAtBottom</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToMinimumContentsLength</enum>
             </property>
             <property name="minimumContentsLength">
              <number>0</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="PlotWidget" name="graphPlot" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1280</width>
          <height>991</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>928</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Export_date">
     <property name="title">
      <string>&amp;Export data</string>
     </property>
     <addaction name="actionPoint_csv"/>
     <addaction name="actionGraph_png"/>
    </widget>
    <addaction name="importCsv"/>
    <addaction name="menu_Export_date"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help ?</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="importCsv">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionPoint_csv">
   <property name="text">
    <string>&amp;Point -&gt; *.csv</string>
   </property>
  </action>
  <action name="actionGraph_png">
   <property name="text">
    <string>&amp;Graph -&gt; *.png</string>
   </property>
  </action>
  <action name="actionBefore_Selected">
   <property name="text">
    <string>&amp;Before selected</string>
   </property>
  </action>
  <action name="actionAfter_selected">
   <property name="text">
    <string>&amp;After selected</string>
   </property>
  </action>
  <action name="action_Delete_Point">
   <property name="text">
    <string>&amp;Delete Point</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ids</tabstop>
  <tabstop>plotBtn</tabstop>
  <tabstop>Gate_Tab</tabstop>
  <tabstop>Gate_P1_x</tabstop>
  <tabstop>Gate_P1_y</tabstop>
  <tabstop>tb_id</tabstop>
  <tabstop>Gate_P2_x</tabstop>
  <tabstop>Table_Point</tabstop>
  <tabstop>Gate_P2_y</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
